lib/astarte_core/triggers/policy_protobuf/tagged_keyword_error.proto

```diff
@@ -0,0 +1,21 @@
+//
+// This file is part of Astarte.
+//
+// Copyright 2021 Ispirata Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+message TaggedKeywordError {
+    int32 keyword = 1;
```

Comment:
This seems a good place for an enum field, see for example `DeviceEventType` in `device_trigger.proto`